src/main/kotlin/me/zeroeightsix/kami/feature/module/Aura.kt

```diff
@@ -49,6 +51,9 @@ object Aura : Module() {
         step = java.lang.Double.MIN_VALUE
     ) Int = 3
 
+    @Setting(name = "Only use Weapon")
```

Comment:
this name seems redundant. it only changes the formatting into something which is inconsistent with the normal naming

Reply:
Its the same thing for "Only use 32K"

Reply:
but not the same as settings from different modules. im not sure what would be right here